sdv/constraints/tabular.py

```diff
@@ -235,6 +235,19 @@ class FixedCombinations(Constraint):
     _combinations_to_uuids = None
     _uuids_to_combinations = None
 
+    @staticmethod
+    def _validate_metadata_specific_to_constraint(metadata, **kwargs):
+        invalid_columns = []
+        column_names = kwargs.get('column_names')
+        for column in column_names:
+            if metadata.columns[column]['sdtype'] not in ['boolean', 'categorical']:
+                invalid_columns.append(column)
+        if invalid_columns:
```

Comment:
Can we have a breakline before this line ? There is still not a plugin to capture this unfortunately, but a rule of thumb is: If the code is going back a level have a breakline.